tests/workflow/orchestrators/test_random.py

```diff
@@ -62,8 +61,7 @@ async def test_process_input(orchestrator_config):
         assert result["content"] == task
 
 
-@pytest.mark.asyncio
-async def test_select_random_speaker(orchestrator_config):
+def test_select_random_speaker(orchestrator_config):
```

Comment:
I went ahead and made they sync instead of async as they were before  bc when I made the signal handling tests I kept getting that the event loop over here was closed. These tests don't need an event loop, so they can just be sync since they are simpler init tests.

Reply:
@Cyb3rWard0g can you advise if this might be related to a possible regression?

Reply:
i reverted this. I was seeing event loop issues in adding the other test, but I think it was just me doing something wrong before causing an err 🤷‍♀️ it works fine reverting back 😅